[enterprise-metrics] Version bump and deprecation in favour of mimir-distributed. (#1228)
grafana/helm-charts: charts/enterprise-metrics/CHANGELOG.md

Entries should include a reference to the Pull Request that introduced the change.

## 1.9.0

> * [CHANGE] This chart is now deprecated and will no longer be updated. Grafana Enterprise Metrics v2.0.0 is included in the new `mimir-distributed` chart which implements Grafana Enterprise Metrics as an option (`enterprise.enabled: true`). Please consult the [Grafana Enterprise Migration Guide](https://grafana.com/docs/enterprise-metrics/latest/migrating-from-gem-1.7/) about how to upgrade to using the new chart.

Reviewer: ```suggestion
* [CHANGE] This chart is now deprecated and will no longer be updated. Grafana Enterprise Metrics v2.0.0 is included in the new `mimir-distributed` chart which implements Grafana Enterprise Metrics as an option (`enterprise.enabled: true`). To upgrade to using the new chart, see the [Grafana Enterprise Metrics migration guide](https://grafana.com/docs/enterprise-metrics/latest/migrating-from-gem-1.7/).
```

Reviewer: I changed the name of this guide to match the product (GEM, rather than GE) and conformed to style (sentence case).
